qt/scientific_interfaces/ISISReflectometry/GUI/Options/IOptionsDialogPresenter.h

```diff
@@ -0,0 +1,37 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_ISISREFLECTOMETRY_IOPTIONSDIALOGPRESENTER_H
+#define MANTID_ISISREFLECTOMETRY_IOPTIONSDIALOGPRESENTER_H
+
+#include "Common/DllConfig.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace ISISReflectometry {
+
+class MANTIDQT_ISISREFLECTOMETRY_DLL OptionsDialogMainWindowSubscriber {
```

Comment:
I'm confused by `MainWindow` in the name here. Would `OptionsDialogPresenterSubscriber` be a better name?

Reply:
Yes, I think it would. Got a bit confused myself to be honest!